pkg/apis/middleware/session.go

```diff
@@ -44,9 +43,9 @@ func CreateTokenToSessionFunc(verify VerifyFunc) TokenToSessionFunc {
 		// Ensure email is verified
 		// If the email is not verified, return an error
 		// If the email_verified claim is missing, assume it is verified
-		if !ptr.Deref(claims.Verified, true) {
-			return nil, fmt.Errorf("email in id_token (%s) isn't verified", claims.Email)
-		}
+		// if !ptr.Deref(claims.Verified, true) {
+		// 	return nil, fmt.Errorf("email in id_token (%s) isn't verified", claims.Email)
+		// }
```

Comment:
just removing this isn't a solution. other users require email verification. you cannot just remove this...